src/makeRobot.js

```diff
@@ -38,7 +38,86 @@
  * @return {Robot}
  */
 function makeRobot(name, wheels, version) {
-  // write code here
+  return {
+    name,
+    wheels,
+    version,
+
+    get info() {
+      // eslint-disable-next-line max-len
+      return `name: ${this.name}, chip version: ${this.version}, wheels: ${this.wheels}`;
+    },
+
+    coords: {
+      x: 0,
+      y: 0,
+    },
+
+    get location() {
+      return `${this.name}: x=${this.coords.x}, y=${this.coords.y}`;
+    },
+
+    goForward(step) {
+      if (step <= 0) {
+        return this;
+      }
+
+      if (!step) {
+        this.coords.y += 1;
+      } else {
+        this.coords.y += step;
+      }
+
+      return this;
+    },
```

Comment:
```suggestion
    goForward(step = 1) {
      if (step > 0) {
        this.coords.y += step;
      }

      return this;
    },
```
You can set the default step. Also, avoid extra checks. Just check if the step is positive and add it to the coord in that case. You will return `this` anyway